### DISCUSSION TITLE
Change the menu and shutting down screen

### DISCUSSION THREAD

**Discussion by FernandoBarcia** [Rparadise-Team/Koriki#8] [Q&A]
Hi! I would like to ask how I can change the shutting down screen and the Koriki settings menu. I can change the bootscreen and others but I cannot find the path to change those.

**Comment by eduardofilo** [comment]
Hi. Those screens are not PNGs, they are drawn by SDL code by the application itself, so they can only be modified by changing the SimpleMenu code and compiling.

**Reply by FernandoBarcia** [reply]
Oh, ok, thanks for the quick reply. There are plans to make it customizable or, maybe, add differents languages to it?

**Reply by eduardofilo** [reply]
Sorry, I don't particularly have interest in making such changes, but maybe someone else does have the motivation.
